modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -221,68 +224,16 @@ class NDB_Form_User_Accounts extends NDB_Form
 
         // multi-site UPDATE
         $uid = $user->getData('ID');
-
-        $us_curr_sites =array();
-        $us_prev_sites =array();
-        //get sites where user is already at to compare
-        $prev_sites = $DB->pselect(
-            "           SELECT CenterID
-                        FROM user_psc_rel
-                        WHERE UserID=:uid",
-            array(
-                "uid" => $uid,
-            )
-        );
-
-        foreach ($prev_sites as $row=>$center) {
-            $pr_cid = $center['CenterID'];
-            array_push($us_prev_sites, $pr_cid);
-        }
-        $curr_sites = $values['CenterIDs'];
-        foreach ($curr_sites as $key=>$val) {
-            //get MultiSelected CenterID
-            array_push($us_curr_sites, $val);
-            //Check if CenterID already in user_psc_rel for that user
-            $result = $DB->pselectRow(
-                "         SELECT CenterID
-                          FROM user_psc_rel
-                          WHERE UserID=:uid",
+        $DB->delete('user_psc_rel', array("UserID" => $uid));
+        foreach ($curr_sites as $site) {
```

Comment:
@xlecours 
this is not working now because $curr_sites is no longer defined (it was defined in the removed line: 241).